uaclient/apt.py

```diff
@@ -67,7 +67,9 @@ def add_auth_apt_repo(repo_filename, repo_url, credentials, suites,
         raise InvalidAPTCredentialsError(
             'Invalid APT credentials provided for %s' % repo_url)
 
-    updates_enabled = True   # TODO determine from apt-cache policy
+    # Does this system have updates suite enabled?
+    policy, _err = util.subp(['apt-cache', 'policy'])
+    updates_enabled = bool(' %s-updates/' % series in policy)
```

Comment:
Question I have here is should we also consider  <series>-proposed as a valid state where we should setup esm trusty-updates suite?

Reply:
You mean if the server ever sends down a %s-proposed line? No.

Reply:
no the server will never send proposed and we should never be adding proposed